ca/ca.go

```diff
@@ -449,7 +455,14 @@ func (ca *certificateAuthorityImpl) issuePrecertificateInner(ctx context.Context
 		NotAfter:          validity.NotAfter,
 	}
 
-	certDER, err := issuer.Issue(req)
+	_, issuanceToken, err := issuer.Prepare(req)
+	if err != nil {
+		ca.log.AuditErrf("Preparing precert failed: serial=[%s] regID=[%d] names=[%s] err=[%v]",
+			serialHex, issueReq.RegistrationID, strings.Join(csr.DNSNames, ", "), err)
+		return nil, nil, nil, berrors.InternalServerError("failed to sign precertificate: %s", err)
```

Comment:
Do we want this to be a slightly different error message from line 470? "Failed to prepare precertificate"?